packages/core/README.md

```diff
@@ -18,9 +16,9 @@ This repository contains the static HTML, Javascript, CSS, and images content of
 
 -   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding help, etc.)
 
-## Use a custom domain
+## Manual deployment using custom domain (CNAME)
 
-To use your custom domain, please put it in a file named `CNAME` inside the `scripts` folder of your local clone of deriv-app.
+For manual deployment, it's necessary to use the file named `CNAME`, please create `scripts/CNAME.txt` and put a custom domain to this file. It's only for the deployment on GitHub Pages for your forked repo of deriv-app.
 
 ## How to work with this project
 
```

Comment:
you can add some description here as well:
To run and work on this workspace you need to use `npm run serve core` command